stable/registry-creds-handler/values.yaml

```diff
@@ -0,0 +1,30 @@
+nameOverride:
+fullnameOverride:
+
+name: registry-creds-handler
+
+logLevel: DEBUG
```

Comment:
canonize it under the a configuration structure that will be passed as a config map configuration to the app

and comment out the options
```yaml
appConfig:
  logLevel: DEBUG
```

Reply:
not sure why we need config map.
I put the required args under `registry` as you suggested and extra args can be passed under `extraArgs`